README.md

```diff
@@ -1,14 +1,13 @@
 # Uniswap Examples
 
-This repository contains examples of applications integrating with Uniswap.
+This repository contains example integrations with Uniswap SDK's and various Web3 environments. More detailed guides walking through the core parts of an example can be found on the [Uniswap docs site](https://docs.uniswap.org/).
```

Comment:
*in* various web3 environements? We should call out that it's just SDK for now, but could include more shortly.